### ISSUE TITLE
Project deployment failed .It seems that goalize package has a problem.

### ISSUE THREAD

**Issue by kumidieat** [SkyZH/goalize#3]
Could you tell me the correct way to deploy it to firebase?

**Comment by kumidieat** [comment]
https://i.loli.net/2017/10/29/59f5f32745b3d.jpg
This is the error message.Please check it out.

**Comment by skyzh** [comment]
Sorry but I couldn’t figure out the exact cause of your situation. I could
only give you following suggestions:

Have you run 'npm install’ before you try building goalize? Please have a
check whether all dependencies were installed successfully.

In the screenshot you provide, I see you are using MinGW console for
building this project. Please make sure node and npm are installed in your
MinGW environment, or you may use Command Prompt, PowerShell or Bash on
Windows. By the way, MinGW doesn't provide full support for Win32 API, thus
you may encounter problems while installing dependencies or building this
project with webpack.

If you have further issues, you can directly reply in this thread.

**Comment by kumidieat** [comment]
https://i.loli.net/2017/11/03/59fc66391e3b4.png
It does not work.I can not finger out what the problem is.

**Comment by skyzh** [comment]
That's all right. It's normal to see so many warnings during installation. However, if you see ERRORS after the command terminates, there might exist other problems.

**Comment by kumidieat** [comment]
ERR messages.
https://ooo.0o0.ooo/2017/11/03/59fc7dd18450e.png

**Comment by kumidieat** [comment]
ERRs have been repeated last time.

**Comment by kumidieat** [comment]
https://ooo.0o0.ooo/2017/11/03/59fc7ee9b9eec.png

**Comment by skyzh** [comment]
Thank you for your reply. As shows in your screenshot, webdriver update
errors can be solved by using a proxy so that updater can fetch drivers. By
the way, you can directly paste image to your reply instead of using an
image provider.

**Comment by kumidieat** [comment]
![r 7 q 6uwbp2i 1s00 5](https://user-images.githubusercontent.com/33200766/32402496-61e0fd14-c160-11e7-9cef-576b8b8d8f11.png)

**Comment by kumidieat** [comment]
![b xr qfynth svb4 iw50 m](https://user-images.githubusercontent.com/33200766/32402516-c9f0f76a-c160-11e7-95e2-054b8d1522ff.png)

**Comment by kumidieat** [comment]
![82ae1 293s oqekzk sh](https://user-images.githubusercontent.com/33200766/32402528-0766d04c-c161-11e7-9cd2-6a2ce0d32df7.png)
still loading

**Comment by skyzh** [comment]
Refresh and paste logs from your Edge console please

**Comment by kumidieat** [comment]
Shall I  deploy it to firebase now?

**Comment by skyzh** [comment]
Before you deploy it to firebase, you must:
* Create a firebase account and create new project, then enable login and change database permission in your firebase console.
* Change settings in your cloned files to your own firebase project here https://github.com/SkyZH/goalize/blob/master/src/app/config/firebase.ts
* Install firebase-cli, compile this project and deploy all static resources to firebase.

Nevertheless, it seems that you haven't yet compiled goalize successfully. Please check if any dependencies fail to install and run `npm run build` again, until `/dist` folder containing all files necessary for deployment appears.

**Comment by kumidieat** [comment]
_nih_

**Comment by kumidieat** [comment]
**npm**

**Comment by kumidieat** [comment]
`npm`
